packages/core/admin/ee/admin/pages/Roles/CreatePage/index.js

```diff
@@ -1,36 +1,52 @@
-import React, { useState, useRef } from 'react';
-import { Header } from '@buffetjs/custom';
-import { Padded } from '@buffetjs/core';
-import moment from 'moment';
-import { Formik } from 'formik';
-import { get, isEmpty } from 'lodash';
-import { useIntl } from 'react-intl';
-import { HeaderLayout, Button } from '@strapi/parts';
-import { AddIcon, EditIcon } from '@strapi/icons';
 import {
-  BaselineAlignment,
   CheckPagePermissions,
   request,
-  useTracking,
   useNotification,
   useOverlayBlocker,
+  useTracking,
 } from '@strapi/helper-plugin';
+import {
```

Comment:
I would love to see at least one test for this component? Maybe the most basic one that makes sure the component can be rendered without errors.